main.css

```diff
@@ -256,7 +263,7 @@ body {
   color: #f4f5f7;
 }
 
-@media screen and (max-width: 950px) {
+@media screen and (max-width: 500px) {
   .headline-1 p {
```

Comment:
## 1.
### Your Desktop Version
![side-white-desktop (copy)](https://user-images.githubusercontent.com/93039359/177644638-0e355ac9-40e9-439d-99b3-e84717bb0914.png)

### Your Mobile Version
![mobile-margin (copy)](https://user-images.githubusercontent.com/93039359/177644659-692c02b3-1b47-40ba-bcb6-d54f28e5385c.png)

- [ ] In this version of mobile, you can see that you have white space on the right. Please remove that white space because your project looks very unprofessional when it has unnecessary white spaces. :blush:


## 2.
### Image 1
![messed-up-design (copy)](https://user-images.githubusercontent.com/93039359/177645878-697d5f4d-3b91-4b58-9dc2-8f57aa97fad0.png)

### Image 2
![cards-not-centered (copy)](https://user-images.githubusercontent.com/93039359/177645919-6a0ddbf4-d835-43c6-a1be-3a36f5096049.png)

- [ ] You can see that the green boxes are the correct positions of the respective content but the contents are going out of the green box. Please fix it. :blush: then your project will look professional. :+1: